src/hooks/useRouterTradeExactIn.ts

```diff
@@ -0,0 +1,35 @@
+import { skipToken } from '@reduxjs/toolkit/dist/query/react'
+import { Currency, CurrencyAmount } from '@uniswap/sdk-core'
+import { useGetQuoteQuery } from 'state/routing/slice'
+import { useActiveWeb3React } from './web3'
+
+const DEFAULT_SLIPPAGE_TOLERANCE = '5'
+const DEFAULT_DEADLINE = '360'
+
+export function useRouterTradeExactIn(amountIn?: CurrencyAmount<Currency>, currencyOut?: Currency) {
+  const { account } = useActiveWeb3React()
+
+  const { isLoading, isError, data } = useGetQuoteQuery(
+    amountIn?.currency?.wrapped?.address &&
+      currencyOut?.wrapped?.address &&
+      amountIn?.currency?.chainId &&
+      currencyOut?.chainId &&
+      account &&
+      amountIn !== undefined
+      ? {
+          tokenIn: { address: amountIn.currency.wrapped.address, chainId: amountIn.currency.chainId },
+          tokenOut: {
+            address: currencyOut.wrapped.address,
+            chainId: currencyOut.chainId,
+          },
+          amount: amountIn.toSignificant(6),
```

Comment:
`.quotient.toExact()` i'm pretty sure amount is supposed to be without decimals

Reply:
@willpote can you confirm?

Reply:
using  `.quotient.toString()`